projects/win32/tomviz.bundle.cmake

```diff
@@ -54,15 +58,8 @@ install(DIRECTORY "${install_location}/lib/tomviz"
 if(itk_ENABLED)
 install(DIRECTORY "${install_location}/lib/itk"
         DESTINATION "lib"
-	USE_SOURCE_PERMISSIONS
-	COMPONENT ${AppName})
-endif()
-
-#------------------------------------------------------------------------------
-set (CPACK_NSIS_MUI_ICON "${CMAKE_CURRENT_LIST_DIR}/InstallerIcon.ico")
```

Comment:
They have been moved to the NSIS specific part of [cpack_config_file.cmake.in](https://github.com/cjh1/tomviz-superbuild/blob/wix-support/projects/win32/cpack_config_file.cmake.in#L7-L11)